website/moneybirdsynchronization/services.py

```diff
@@ -0,0 +1,249 @@
+    except Administration.Error as e:
+        emails.send_sync_error(e, food_order.moneybird_external_invoice)
+
+
+def register_contribution_payment(instance):
+    if settings.MONEYBIRD_SYNC_ENABLED is False:
+        return
+
+    apiservice = get_moneybird_api_service()
+
+    contact_id = apiservice.get_contact_id_by_customer(instance)
+
+    invoice_info = apiservice.create_external_sales_info(
+        contact_id, instance.payment, contribution=True
+    )
+
+    try:
+        response = apiservice.create_external_sales_invoice(invoice_info)
+        instance.moneybird_external_invoice.moneybird_invoice_id = response["id"]
+        instance.moneybird_external_invoice.save()
+    except Administration.Error as e:
+        emails.send_sync_error(e, instance.moneybird_external_invoice)
+
+
+def delete_external_invoice(instance):
+    if settings.MONEYBIRD_SYNC_ENABLED is False:
+        return
+
+    apiservice = get_moneybird_api_service()
+    apiservice.delete_external_invoice(instance.moneybird_external_invoice)
+
+
+def sync_contacts():
+    if settings.MONEYBIRD_SYNC_ENABLED is False:
+        return
+
+    apiservice = get_moneybird_api_service()
+
+    unsynced_members = Member.objects.filter(
+        ~Q(
+            id__in=Subquery(
+                MoneybirdContact.objects.filter(member=OuterRef("pk")).values("member")
+            )
+        )
+    )
+
+    for member in unsynced_members:
+        contact = MoneybirdContact(member=member)
+        contact.save()
+
+    # fetch contact ids from moneybird
+    api_response = apiservice.api.get("contacts")
+
+    # fetch contact ids from contact model
+    contact_info = [
+        contact.get_moneybird_info() for contact in MoneybirdContact.objects.all()
+    ]
+
+    # find contacts in contact model that are not in moneybird and add to moneybird
+    moneybird_ids = [int(info["id"]) for info in api_response]
+    for contact in contact_info:
+        if contact["id"] is None or int(contact["id"]) not in moneybird_ids:
+            contact = MoneybirdContact.objects.get(
+                member=Member.objects.get(pk=contact["pk"])
+            )
+            response = apiservice.add_contact(contact)
+            contact.moneybird_id = response["id"]
+            contact.moneybird_version = response["version"]
+            contact.save()
+
+    moneybird_info = []
+    for contact in api_response:
+        if len(contact["custom_fields"]) > 0:
+            moneybird_info.append(
+                {
+                    "id": contact["id"],
+                    "version": contact["version"],
+                    "pk": contact["custom_fields"][0]["value"],
+                }
+            )
+
+    ids = [info["id"] for info in contact_info]
+    for moneybird in moneybird_info:
+        if int(moneybird["id"]) not in ids:
+            apiservice.delete_contact(moneybird["id"])
+
+
+def sync_statements():
+    if settings.MONEYBIRD_SYNC_ENABLED is False:
```

Comment:
```suggestion
    if not settings.MONEYBIRD_SYNC_ENABLED:
```
If `MONEYBIRD_SYNC_ENABLED=None`, moneybird sync should be disabled.

Reply:
@nvoers can you change this everywhere